app/adapters/application.js

```diff
@@ -9,9 +9,17 @@ const { JSONAPIAdapter } = DS;
 import { pluralize } from 'ember-inflector';
 
 export default JSONAPIAdapter.extend(DataAdapterMixin, {
-  authorizer: 'authorizer:oauth-2',
   host: ENV.api.hostname,
 
+  headers: computed('session.data.authenticated.access_token', function() {
+    const headers = {};
```

Comment:
Ik zou de headers liever gemerged zien dan dat we ze helemaal nieuw construeren...

Reply:
Aan de andere kant ga je hier waarschijnlijk niet bij de headers kunnen komen, denk ik. Of misschien is dit het startpunt voor Ember om headers te bouwen, dat andere dingen later hierbij ingemerged worden wanneer nodig

Reply:
ik denk inderdaad dat dit het startpunt was. Dit was de aanbevolen methode van ember-simple-auth (https://github.com/simplabs/ember-simple-auth#walkthrough , iets omlaag scrollen)